instrumentation/opentelemetry-instrumentation-tornado/src/opentelemetry/instrumentation/tornado/__init__.py

```diff
@@ -71,7 +72,17 @@ def get_excluded_urls():
     return ExcludeList(urls)
 
 
+def get_traced_request_attrs():
+    attrs = configuration.Configuration().TORNADO_TRACED_REQUEST_ATTRS or ""
+    if attrs:
+        attrs = [attr.strip() for attr in attrs.split(",")]
+    else:
+        attrs = []
+    return attrs
```

Comment:
is it worth moving this code into the utils package as well since it seems likely to be repeated?

Reply:
Agree. I think it might be better to have convenience methods on the configuration class itself for common config items. Something like,

```python
cfg = configuration.Configuration()
cfg.traced_methods("django")
cfg.excluded_urls("django")
```

Reply:
Created an issue for it. https://github.com/open-telemetry/opentelemetry-python/issues/1217